Example/SmileID.xcodeproj/project.pbxproj

```diff
@@ -670,15 +670,15 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
-				CURRENT_PROJECT_VERSION = 23;
+				CURRENT_PROJECT_VERSION = 26;
```

Comment:
Does this file need to be committed?

Reply:
Yep. Xcode maintains a lot of generated files that do need to be committed

Reply:
This change here is bumping the build number for the updated TestFlight release

Reply:
Aah

Thanks